airship/kubeadm-airship-gap-analysis.md

```diff
@@ -13,6 +13,59 @@ Jira Issues:
 Jira Issues:
 - [CA rotation](https://airship.atlassian.net/browse/AIR-138)
 
+ 
+## Use pre-generated certs and user provided CA
+
+This is possible with Kubeadm. We can provide CA certs only or we can provide the whole list of certificates in ```/etc/kubernetes/pki``` . In case we provide CA cert only, kubeadm will generate the other certificates using the CA cert. In the latter case, kubeadm will take the pre-generated certs and avoid generating new ones. The placement of the certs can be done using a script.
+
+In case of CABPK following certificates and keys can be specified by the user:
+```
+cluster-ca-cert
+cluster-ca-key
+    
+etcd-ca-cert  
+etcd-ca-key
+    
+front-proxy-ca-cert
+front-proxy-ca-key
+
+service-account-private-key
+service-account-public-key
+```
+
+## Rotating CA 
+
+Kubeadm currently has an option to renew the CA certificates manually, based on a new CA certificate placed using the method described in previous comment. It is however an alpha version command - 
```

Comment:
automatically

Reply:
Not sure what do you mean

Reply:
Kubeadm currently has an option to renew the CA certificates automatically